globals/types.ts

```diff
@@ -61,8 +61,8 @@ export interface Review {
 	downvotes: number
 	/* --- general review data --- */
 	workload: number
-	difficulty: TRatingScale
-	overall: TRatingScale
+	difficulty: TRatingScale | 0
+	overall: TRatingScale | 0
```

Comment:
let's change this a bit and move this out of `Course` into the component...this is technically not semantically correct; `Course` should be in the post-submit form. That was a bad suggestion/fix on my part, sorry about that!